db/schema.rb

```diff
@@ -225,6 +225,16 @@
     t.index ["location_id"], name: "index_ndcs_on_location_id"
   end
 
+  create_table "wb_extra_country_data", force: :cascade do |t|
+    t.bigint "location_id"
+    t.string "year"
+    t.string "GDP"
```

Comment:
Please use all lowercase letters for consistency all with other tables (even though gdp is an acronym)